_posts/2020-11-23-puppetize-digital.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Puppetize Digital 2020 - Content Selection"
+author: dev_el_ops
+categories:
+  - puppet
+  - conference
+tags:
+  - content
+  - demos
+  - tips-and-tricks
+  - videos
+---
+
+Last week saw [Puppetize Digital 2020](https://digital.puppetize.com/) our annual conference on all things Puppet.
+If you've missed something as it was streamed, over the weekend the production team uploaded all the talks to [the Puppet YouTube channel](https://www.youtube.com/user/PuppetLabsInc/videos).
+There is a [playlist](https://www.youtube.com/playlist?list=PLV86BgbREluXpEPnW_Bb4kgZAlGit4EnW) with all videos, and in this post I'll directly link to my favourites - of course focused on Puppet Content and the IAC team's work.
+
+## Puppet Pair Programming with Visual Studio Code
+
+The [Puppet VSCode Plugin](https://puppet-vscode.github.io/) has evolved into THE canonical way to edit puppet modules.
+The authors of the plugin, [James Pogran](https://www.linkedin.com/in/jamespogran/) and [Glenn Sarti](https://www.linkedin.com/in/glenn-sarti/) are sharing the newest updates to the plugin, as well as some of the benefits of working with a modern, full-featured Development Environment:
+
+{% youtube j3o-U1c_dXs 800 450 %}
+
+Since last week we also provide a default [gitpod](https://gitpod.io) and [codespaces](https://github.com/features/codespaces) environment on our supported modules. So either install the [gitpod browser extension]()
```

Comment:
Looks like something is missing here?

Reply:
I was looking exactly at the same thing on the blog